android/assets/jsons/translations/template.properties

```diff
@@ -509,8 +509,9 @@ Turn =
 turns = 
 turn = 
 Next unit = 
-Pick a policy = 
-Movement = 
+Fog of War =
+Pick a policy =
```

Comment:
You removed the spaces after the = 0_0
Those are super important, also for the Fog of War

Reply:
Sorry! -_-
I believe it is IDE somehow autocorrected it. Fixed it.